examples/convert_little_endian_to_big.cpp

```diff
@@ -30,10 +30,10 @@ int main()
     std::cout << "in is: " << std::hex << in << std::endl;
     // If the host is little endian, the put function
     // should change the byte order (no change for big endian)
-    endian::convert_endian::put32(in, data);
+    endian::little_endian::put32(in, data);
```

Comment:
These comments are not correct right? I the host is tittle endian then the `little_endian::put` should not do anything - right?